<!-- Aruco marker localizer launch. For usage in start_recognizers or to be called by standalone launch file. @author Pascal Meissner, Tobias Allgeyer. @version See SVN -->

<launch>

	<env name="ROSCONSOLE_CONFIG_FILE" 
	  value="$(find asr_aruco_marker_recognition)/log/rosconsole.config"/>
  	<node name="asr_aruco_marker_recognition" pkg="asr_aruco_marker_recognition" type="asr_aruco_marker_recognition" output="screen">
	    
	   <rosparam file="$(find asr_aruco_marker_recognition)/launch/aruco_marker_recognition_params.yaml" />
	   
	   <param name="showThresholdImage" value="False"/>
	   <param name="thresholdMethod" value="1"/>
	   <param name="gaussianBlurSigma" value="3"/>
	   
	   <param name="sharpenWeightImage" value="1.7"/>
	   <param name="sharpenWeightBlur" value="-0.7"/>
	   <param name="minSize" value="0.01"/>
	   <param name="maxSize" value="0.5"/>
	   <param name="minThreshold" value="7"/>
	   <param name="maxThreshold" value="7"/>
	   
  	</node>
  
</launch>
